src/regularTasks.ts

```diff
@@ -245,31 +264,33 @@ const GLOBAL_TASKS: CandyTask[] = [
     completed: () => have($effect`Everything Looks Red`),
     do: () => adv1(wanderWhere("backup"), -1, ""),
     sobriety: "sober",
-    canInitializeDigitize: true,
+    post: () => (digitizeInitialized = true),
     outfit: combatOutfit,
     combat: new CandyStrategy(Macro.skill($skill`Free-For-All`)),
   },
   {
     name: "Nemesis Assassin",
     completed: () => Counter.get("Nemesis Assassin window end") > 0,
     do: () => adv1(wanderWhere("wanderer"), -1, ""),
-    canInitializeDigitize: true,
+    post: () => (digitizeInitialized = true),
     outfit: combatOutfit,
     combat: new CandyStrategy(),
   },
   {
     name: "Initialize Digitize",
-    completed: () => CandyState.digitizeInitialized,
+    completed: () => digitizeInitialized,
     do: (): void => {
-      CandyState.runSource?.prepare();
+      runSource?.prepare();
       wanderWhere("backup");
     },
-    canInitializeDigitize: true,
-    post: () => (CandyState.runSource = null),
+    post: (): void => {
+      digitizeInitialized = true;
+      runSource = null;
+    },
     outfit: (): Outfit => {
-      CandyState.initializeRunSource();
-      const req = CandyState.runSource?.constraints?.equipmentRequirements?.();
-      const familiar = CandyState.runSource?.constraints?.familiar?.();
+      initializeRunSource();
```

Comment:
maybe we should put this in a getter rather than relying on the knowledge that outfit is the first thing called that actually uses the runSource; so `function getRunSource(): ActionSource { return runSource ??= all that stuff }`